kubeweekly/values.yaml

```diff
@@ -1,23 +1,38 @@
----
-# default values supplied for templates/* files
-job:
-  name: example-app
-  namespace: default
-  image: "brndnmtthws/nginx-echo-headers"
-  schedule: "*/12 * * * 3"
-  initContainers:
-    name: kubeweekly-scrapper
-    image: kubeweekly-scrapper:latest
-    args: npm run trigger
+jobLimit: 1
 
+scrapper:
+  name: kubeweekly-scrapper-jobs
+  namespace: kubeweekly
+  image: "cloudnativeid/kubeweekly:1.0.0"
+  schedule: "0 12 * * *"
+  args: "npm run trigger"
   env:
```

Comment:
this env is a bit too verbose, values.yaml should be simple and easy to configure at high level, get inspiration from upstream helm chart repo on how values.yaml usually look like